stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -105,7 +108,14 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
         modules.add(new Console());
         modules.add(new CSS());
         modules.add(new Debugger());
-        modules.add(new DOM());
+
+        modules.add(new DOM(new DOMProvider.Factory() {
```

Comment:
This code should continue to read as one-liners.  It's intended as an extensibility point and should be terse.  Prefer:
```java
modules.add(new DOM(new AndroidDOMProviderFactory()));
```